pkg/api/validation/validation.go

```diff
@@ -2844,10 +2799,16 @@ func ValidateNodeUpdate(node, oldNode *api.Node) field.ErrorList {
 	// Clear status
 	oldNode.Status = node.Status
 
+	// update taints
```

Comment:
> @liggitt would it be OK for us to do that in this PR?

Adding tolerations is no different than mutating them, security-wise (preventing user-specified tolerations is a key way to keep user pods off nodes). I'd prefer to defer the discussion about pod-spec mutation to after 1.6